<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoWavetablesWindow</class>
 <widget class="QMainWindow" name="DemoWavetablesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>103</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select wave</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Select quality</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Play sweep</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="valWave"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="valQuality"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btnPlaySweep">
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="icon">
       <iconset theme="media-playback-start">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
